lib/SIL/Verifier/SILVerifier.cpp

```diff
@@ -7298,10 +7810,12 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
         auto *actorProtocol = ctx.getProtocol(KnownProtocolKind::Actor);
         auto *distributedProtocol =
             ctx.getProtocol(KnownProtocolKind::DistributedActor);
-        require(argType->canBeIsolatedTo() ||
-                    genericSig->requiresProtocol(argType, actorProtocol) ||
-                    genericSig->requiresProtocol(argType, distributedProtocol),
-                "Only any actor types can be isolated");
+        require(
```

Comment:
@gottesmm looks like you originally added the generic signature logic in https://github.com/swiftlang/swift/pull/73602 – would you mind taking a look please?

Reply:
@gottesmm and on a tangential note... why is this function named `verifyParentFunctionSILFunctionType`?